.travis.yml

```diff
@@ -1,5 +1,5 @@
-dist: trusty
-sudo: false
+dist: xenial
```

Comment:
Please use the same `dist` and `sudo` like in https://github.com/doctrine/orm/blob/2.8.x/.travis.yml

Reply:
`sudo` is deprecated, and eventually will start erroring. It should be removed from config files.

Reply:
I didn't know that. So it can stay removed from the travis-ci file.
https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure